external-crates/move/testing-infra/transactional-test-runner/src/framework.rs

```diff
@@ -109,7 +109,7 @@ fn merge_output(left: Option<String>, right: Option<String>) -> Option<String> {
 #[async_trait]
 pub trait MoveTestAdapter<'a>: Sized + Send {
     type ExtraPublishArgs: Send + Parser;
-    type ExtraValueArgs: ParsableValue;
+    type ExtraValueArgs: ParsableValue + Clone;
```

Comment:
This is an interesting type! I assume it complains otherwise why we'd need Clone here?